cytominer_eval/evaluate.py

```diff
@@ -104,6 +107,15 @@ def evaluate(
     enrichment_percentile : float or list of floats, optional
         Only used when `operation='enrichment'`. Determines the percentage of top connections
         used for the enrichment calculation.
+    hitk_percent_list : list or "all"
+        Only used when operation='hitk'
+        A list of percentages at which to calculate the percent scores, ie the amount of indexes below this percentage.
+        If percent_list == "all" a full dict with the length of classes will be created.
+        Percentages are given as integers, ie 50 means 50 %.
+    hitk_group_col : str, optional
+        Only used when operation='hitk'
```

Comment:
in the docstring, please specify what the default value is, explicitly.

It is great to warn users to not deviate from `pair_a_index` (so keep that), I'm just saying to explicitly state that it is the default